sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyVaultKey.java

```diff
@@ -98,32 +96,6 @@ public KeyType getKeyType() {
         return key.getKeyOps();
     }
 
-    /**
-     * Creates an instance of KeyVaultKey
-     * @param keyId the identifier of the key
-     * @param jsonWebKey the json web key to be used for crypto operations
-     * @return The Key Vault Key object.
-     */
-    public static KeyVaultKey fromKeyId(String keyId, JsonWebKey jsonWebKey) {
```

Comment:
Removing public methods from a public package is a breaking change. Why are we removing these methods?

Reply:
This was introduced in beta.